Branch-SDK/src/io/branch/referral/DeviceInfo.java

```diff
@@ -234,19 +194,28 @@ public String getPackageName() {
      * @return {@link String} with app version value
      */
     public String getAppVersion() {
-        return appVersion_;
+        return systemObserver_.getAppVersion(context_);
     }
 
-    public boolean isHardwareIDReal() {
```

Comment:
When we remove this, let's ensure the key/value is still sent on outgoing events in request payload